### PR TITLE
Por 363

### PR DESCRIPTION
**Pull request by wilblack**

### CODE REVIEW

File: mp/urls.py [hunk 1]
```diff
@@ -44,10 +44,11 @@
                         visualize.views.show_embedded_map),
                        (r'^([\w-]*)/catalog/', explore.views.data_catalog),
                        (r'^$', RedirectView.as_view(url='/visualize')),
+                       url(r'^media/(?P<path>.*)$', 'django.views.static.serve', {'document_root': settings.MEDIA_ROOT}),
                        url("^media/admin/(?P<path>.*)$",
                                "django.views.static.serve",
                                {"document_root": settings.ADMIN_MEDIA_PATH}),
-                       (r'', include('madrona.common.urls')),
+                       #(r'', include('madrona.common.urls')),
```
Comment (wilblack) [inline mp/urls.py]: This need to be uncommented again.

File: mp/tests/proxy_counts.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import json
```
Comment (wilblack) [inline mp/tests/proxy_counts.py:1]: This whole file was added in to test the proxy results from the marine debris database.

**Comment by wilblack** [comment]
@rcranley I went through and cleaned-up all the run-time changes i made to get this going locally. So what is on this branch is the fix to remove the plastic bags grocery shopping trash filter and some tests to test the proxy service, and some updated readme. so it should be good to merge.

**Comment by rcranley** [comment]
@wilblack  Looks good.

:+1:

**Pull request merged**